src/web/components/Core/Dialog.scss

```diff
@@ -0,0 +1,68 @@
+.dialog-overlay {
+  background-color: rgba(255, 255, 255, 0.8);
+  position: fixed;
+  inset: 0;
+  animation: overlayShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.dialog-content {
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 0px 4px 14px rgba(0, 0, 0, 0.25);
```

Comment:
Again, use a theme colour. I took a look at Figma and it looks like we've got some (0, 0, 0, 0.15) (which is --theme-box-shadow) and some (0, 0, 0, 0.25). Maybe just set this to --theme-box-shadow and let's check with Kristen if she has a preference, but I expect we can make them all the same.